### ISSUE TITLE
Enable staging ic's for cycled experiments. Part 2: add ice and ocean IC's

### ISSUE THREAD

**Issue by guillaumevernieres** [NOAA-EMC/global-workflow#1204]
Follow up to what @aerorahul  added in [PR#1199](https://github.com/NOAA-EMC/global-workflow/pull/1199)

**Comment by guillaumevernieres** [comment]
@aerorahul , I just realized that this staging option is for "cold" starting only. Is there a plan to extend it to "warm starting" as well?

**Comment by aerorahul** [comment]
@guillaumevernieres 
yes.  At the time, I only had the cold-starts.  Should be easy to add warm starts. 
warm starts a bit trickier because depending on the configuration, a few file types might be present/absent.

I can add warm starts in a couple days.  Do you have some warm starts to share?

**Comment by guillaumevernieres** [comment]
> @guillaumevernieres yes. At the time, I only had the cold-starts. Should be easy to add warm starts. warm starts a bit trickier because depending on the configuration, a few file types might be present/absent.
> 
> I can add warm starts in a couple days. Do you have some warm starts to share?

@aerorahul , we're using restarts from the RT tests:
```
/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/runs/s2s-test/RT-IC/RESTART
```

**Comment by aerorahul** [comment]
@guillaumevernieres 
Thanks!
I'll open a PR with that functionality.

**Comment by WalterKolczynski-NOAA** [comment]
@aerorahul was this completed as part of #1274?

**Comment by aerorahul** [comment]
@WalterKolczynski-NOAA yes.

Closed via #1274
